include/nlohmann/detail/conversions/to_json.hpp

```diff
@@ -435,7 +435,7 @@ inline void to_json(BasicJsonType& j, const T& t)
 template<typename BasicJsonType>
 inline void to_json(BasicJsonType& j, const std_fs::path& p)
 {
-    j = p.string();
+    j = p.u8string();
```

Comment:
Note that from C++20, `u8string` will return `std::u8string` instead of `std::string` - I am not sure if this assignment works in this case.

Reply:
I see the `reinterpret_cast` there only..)

Reply:
I saw a remark on cppreference.com

Reply:
I mean the u8string to string conversion here in that issue